L/LRSLib_PolyhedraFork/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LRS_PolyhedraFork"
```

Comment:
Why this verbose name?  Note that this name must match exactly the name of the directory this file is in